domains/transfer-agreements/TransferAgreements.API/API.IntegrationTests/Testcontainers/WalletContainer.cs

```diff
@@ -31,7 +27,7 @@ public WalletContainer()
                 .WithPortBinding(80, true)
                 .WithCommand("--serve", "--migrate")
                 .WithEnvironment("ConnectionStrings__Database", postgresConnectionString)
-                // - This should be implemented correctly, right now it apparantly doesn't use it
+                // - This should be implemented correctly, right now it apparently doesn't use it
```

Comment:
Why is this here?
If this is something that isnt being used it should be deleted.
If there is some implementation needed we should do it as a TODO with a link to a zenhub issue

Reply:
There is no Zenhub Issue. Should i create one then?